doc/source/user/troubleshooting-importerror.rst

```diff
@@ -146,3 +146,21 @@ following in python::
 
 This may mainly help you if you are not running the python and/or NumPy
 version you are expecting to run.
+
+
+C-API incompatibility
+---------------------------
+
+If you see this error you may have:
```

Comment:
Please add at least some of the text of the error message. Also the lines below should be reflowed to not exceed ~80 columns

Reply:
@mattip Thanks for help, I updated the PR. Is it ok now ?